packages/app/src/app/pages/Pro/legacy-pages/WorkspacePlanSelection.tsx

```diff
@@ -1,591 +1,284 @@
+              }
+              features={PERSONAL_FEATURES}
+              cta={personalProCta}
+              isHighlighted
+            >
+              <Stack gap={1} direction="vertical">
+                <Text size={32} weight="500">
+                  ${user.subscription.amount}
                 </Text>
-              ) : null}
-            </>
+                {user.subscription.duration === 'yearly' ? (
+                  <Text>
+                    charged annually on{' '}
+                    {format(new Date(user.subscription.since), 'MMM dd')}
+                  </Text>
+                ) : (
+                  <Text>
+                    charged on the{' '}
+                    {format(new Date(user.subscription.since), 'do')} of each
+                    month
+                  </Text>
+                )}
+              </Stack>
+            </SubscriptionCard>
           ) : (
-            <Button
-              loading={loading}
-              disabled={
-                // non-admins can't upgrade
-                activeUserAuthorization !== TeamMemberAuthorization.Admin
-              }
-              onClick={() => setStep(Step.InlineCheckout)}
-              css={css({
-                fontSize: 3,
-                height: 10,
-                fontFamily: 'Lato, sans-serif',
-                fontWeight: 700,
-              })}
+            <SubscriptionCard
+              title="Team Pro"
+              features={TEAM_PRO_FEATURES}
+              cta={teamProCta}
+              isHighlighted
             >
-              Continue
-            </Button>
+              <Stack gap={1} direction="vertical">
+                <Text size={32} weight="500">
+                  ${subscription.unitPrice}
+                </Text>
+                <Text>
+                  <div>per editor</div>
+                  {isTeamAdmin ? (
+                    <div>
+                      {getBillingText({
+                        quantity: subscription.quantity,
+                        unitPrice: subscription.unitPrice,
+                        billingInterval: subscription.billingInterval,
+                      })}
+                    </div>
+                  ) : null}
+                </Text>
+              </Stack>
+            </SubscriptionCard>
           )}
         </Stack>
-      )}
-    </div>
-  );
-};
 
-const PlanCard: React.FC<{
-  plan: Plan;
-  billingInterval: Plan['billingInterval'];
-  setBillingInterval: (billingInterval: Plan['billingInterval']) => void;
-  // Omitting new trialEnd, trialStart, cancelAtPeriodEnd and active
-  currentSubscription: Omit<
-    ProSubscription,
-    'trialEnd' | 'trialStart' | 'cancelAtPeriodEnd' | 'active'
-  > | null;
-}> = ({ plan, billingInterval, setBillingInterval, currentSubscription }) => {
-  const isSelected = plan.billingInterval === billingInterval;
-  const isCurrent =
-    plan.billingInterval === currentSubscription?.billingInterval;
+        {isPersonalSpace ? (
```

Comment:
Shouldn't this check if the user is an admin as well?

Reply:
I don't think it existed on the previous iteration for teams, so I left it out here too. Still able to cancel on the `/dashboard/settings` page though, unlike the Patron. Was confused about why this distinction was made, but I guess I'm not the only one. I can add it, it would make sense.

Reply:
Created a task in Linear to address this later.